### PR TITLE
Docs

### PR DESCRIPTION
**Pull request by nevir**
@morethanreal

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -23,7 +23,7 @@
     "paper-styles": "PolymerElements/paper-styles#^0.8.0"
   },
   "devDependencies": {
-    "iron-doc-viewer": "PolymerElements/iron-doc-viewer#^0.8.0",
+    "iron-doc-viewer": "PolymerElements/iron-doc-viewer#^0.8.6",
```
Comment (morethanreal) [inline bower.json]: Shouldn't `^0.8.0` pick up 0.8.6 also?

**Comment by morethanreal** [comment]
lgtm, please squash

File: bower.json (hunk 1 above)
Comment (nevir) [inline bower.json]: Yeah, but I made some backwards incompatible changes in `0.8.6` (`src` is not required in that version and beyond)

**Comment by nevir** [comment]
squashed!

**Pull request merged**